src/build/buildExecutor.ts

```diff
@@ -43,6 +44,9 @@ export class BuildExecutor {
         };
 
         if (!BuildExecutor.skipRestore) {
+            let cacheSize = await getFolderSizeInMB(path.join(os.homedir(), '.docfx'));
+            this.eventStream.post(new BuildCacheSizeCalculated(correlationId, cacheSize));

```

Comment:
should we calculate the cache size after the restore finished? or it will be last time cache folder size

Reply:
yes, I also thought about it... Let's put it afterwards